### ISSUE TITLE
Error: Cannot find module 'arquero'

### ISSUE THREAD

**Issue by integzz** [DonJayamanne/typescript-notebook#33]
Hi, this is an awesome extension. I tried the new release 2.02. However, an error message popped when I tried to import 'arquero'

```console
Error: Cannot find module 'arquero'
Require stack:
- <repl>
node:internal/modules/cjs/loader:936
  throw err;
  ^

    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Function.b._load (/Users/integrator/.vscode/extensions/donjayamanne.typescript-notebook-2.0.2/out/extension/server/index.js:2:115314)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:94:18)
    at <Cell 1> [1, 66]
    at Script.runInContext (node:vm:139:12)
    at Script.runInNewContext (node:vm:144:17)
    at Object.runInNewContext (node:vm:298:38)
    at x (/Users/integrator/.vscode/extensions/donjayamanne.typescript-notebook-2.0.2/out/extension/server/index.js:2:113287)
```

**Comment by integzz** [comment]
BTW, I installed 'arquero' globally. Does the extension only support modules locally ?

**Comment by DonJayamanne** [comment]
Thats weird, i'm able to repro this, I'll fix this tomorrow.
Not sure why its only looking locally, thanks for identifying this.
For now, yes, please do use the local install of the module.
Will fix this tomorrow

**Comment by DonJayamanne** [comment]
Looks like you cannot use `arquero` as a global module.
It doesn't work in regular node.js repl. Hence not a bug in the extension.

<img width="620" alt="Screen Shot 2021-09-06 at 23 07 35" src="https://user-images.githubusercontent.com/1948812/132292743-b9c9ba9c-216a-4c44-97d8-b220ae7851c6.png">

**Comment by DonJayamanne** [comment]
Thanks for reporting this issue, but since i can repro this outside vscode, its not an issue in the extension.
Closing as a result. Sorry I can't help here

**Comment by integzz** [comment]
Thank you all the same.

Sent from my iPhone

On Sep 7, 2021, at 14:08, Don Jayamanne ***@***.***> wrote:

﻿

Closed #33<https://github.com/DonJayamanne/typescript-notebook/issues/33>.
